helpers/cache.js

```diff
@@ -16,9 +16,14 @@ module.exports.connect = function (cacheEnabled, config, logger, cb) {
 		return cb(null, { cacheEnabled: cacheEnabled, client: null });
 	}
 
+	Object.keys(config).forEach(function (key){
```

Comment:
Well atleast theres that. I didn't know before hand when i made the suggestion to do this.